changelog.md

```diff
@@ -3,6 +3,7 @@
 - [BUG] Fix wrong count for `findAndCountAll` with required includes [#4016](https://github.com/sequelize/sequelize/pull/4016)
 - [BUG] Fix problems related to parsing of unique constraint errors [#4017](https://github.com/sequelize/sequelize/issues/4017) and [#4012](https://github.com/sequelize/sequelize/issues/4012)
 - [BUG] Fix postgres path variable being surrounded by quotes to often in unique constraint errors [#4034](https://github.com/sequelize/sequelize/pull/4034)
+- Standardize `findAndCount` arguments - changed `findOptions` to `options`
```

Comment:
Probably don't need a changelog entry for it, not sure the end users care a great deal.